commons/error_messages.py

```diff
@@ -38,9 +38,8 @@
 CORTX_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyOwnedByYou) when calling"
                                 "the CreateBucket operation: Your previous request to create "
                                 "the named bucket succeeded and you already own it.")
-RGW_ERR_COPY_OBJ = ("An error occurred (InvalidRequest) when calling the CopyObject operation:"
-                    " The specified copy source is larger than the maximum allowable size for a "
-                    "copy source: 5368709120")
+RGW_ERR_COPY_OBJ = ("An error occurred (EntityTooLarge) when calling the CopyObject "
```

Comment:
Make necessary changes in s3 tests/libs as well.

Reply:
No change required